android/app/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,7 @@
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
     <application
-        android:name="io.flutter.app.FlutterApplication"
+        android:name=".WHApplication"
```

Comment:
Do we have a reason for changing this?

Reply:
Yes, we need a custom Android App singleton class in order to do native android Sentry initialisation in its `onCreate()`

Reply:
Yeah cool 👍